src/v/net/server.cc

```diff
@@ -151,10 +151,17 @@ void server::print_exceptional_future(
               ctx,
               address,
               ex);
+        } else if (is_message_error(ex)) {
```

Comment:
any suggestions for a better name?

1. `is_app_error`?
2. `is_client_error`?

Reply:
I think my issue with the naming is that if it returns `true` it's a `warn` and not an `error`...

I think `is_client_error` makes sense?  We want to not be too verbose when it's the client misbehaving, like in the auth path.